NUITizenGallery/Examples/NavigatorTest/NavigatorTest4.cs

```diff
@@ -65,7 +65,7 @@ public NavigatorContentPage4(Window win)
                 PushPopNavigator(window);
             };
 
-            //navigator = window.GetDefaultNavigator();
+            //navigator = window.GetDefaultNavigator();
```

Comment:
unnecessary code change.
what about remove this line?

Reply:
thank you for review.
it is changed.